stopes/ui/seamlisten/backend/app/fileviewer.py

```diff
@@ -161,3 +167,26 @@ async def general_query(query: DefaultQuery) -> Response:
         a line with audio file, start and end timestamps
         """,
         )
+
+
+def gather_folder_contents(folder_path, max_depth=5):
+    def gather_contents_recursive(folder_path, current_depth):
+        if current_depth > max_depth:
+            return {"folder": folder_path, "subfolders": [], "audio_files": []}
```

Comment:
it might be better to return "null" here for `audio_files` and `subfolders`, or have a separate flag "unexplored" or something like that. So that we know that it's not that the folder is empty, but that it just wasn't listed. This would enable the UI to make new queries when someone tries to expand this folder.